core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -382,7 +382,7 @@
     private transient SetupWizard setupWizard;
 
     /**
-     * Number of executors of the master node.
+     * Number of executors of the built-in node.
```

Comment:
```suggestion
     * Number of executors on the built-in node.
```
or maybe "for the built-in node"?